Tensile/BenchmarkProblems.py

```diff
@@ -238,7 +237,7 @@ def benchmarkProblemType(problemTypeConfig, problemSizeGroupConfig, problemSizeG
         print1("# Benchmark Step: {} - {} {:.3f}s".format(groupName, stepName, elapsedTime))
         print1("# Num Sizes: {}".format(benchmarkStep.problemSizes.totalProblemSizes))
         print1("# Fork Parameters:")
-        for k, v in benchmarkStep.forkParams.items():
+        for k, v in sorted(benchmarkStep.forkParams.items()):
```

Comment:
I am not familiar with this part of code.
Please give me some more detail about this change.
What is this for?
How your change works better than before?

Reply:
This command sorts the fork parameters in the output, so you can find the parameters easier.

Before my changes Tensile output looks like:
# Fork Parameters:
#     StaggerU: [0, 32]
#     StaggerUStride: [128, 256]
#     StaggerUMapping: [1, 2, 3]
#     MatrixInstruction: [[32, 32, 2, 1], [32, 32, 2, 1, 1, 1, 1, 1, 4]]
#     DepthU: [8, 16]

After my changes (ordered alphabetically): 
# Fork Parameters:
#     DepthU: [8, 16]
#     MatrixInstruction: [[32, 32, 2, 1], [32, 32, 2, 1, 1, 1, 1, 1, 4]]
#     StaggerU: [0, 32]
#     StaggerUMapping: [1, 2, 3]
#     StaggerUStride: [128, 256]

Reply:
OK. I got it.
That looks good.